3725/3/app/views/posts/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="post_item">
+  <div class="post_item_title">
+    <a href="/posts/all/<%= post.id %>"><%= post.title %></a>
+  </div>
+  <div class="post_item_summary">
+    <p><%= post.summary %></p>
+  </div>
+  <div class="post_item_body">
+    <p><%= post.body %></p>
+  </div>
+</div>
+<% if session[:user_id] %>
+<div class="reviews">
+  <div class="reviws_title">
+    <p>You can leave your review</p>
+  </div>
+  <div class="reviews_form">
+    <%= erb :'posts/form_review.html' %>
+  </div>
+</div>
+<% else %>
+<div class="reviws_title">
+  <p><a href ="../../users/login">log in</a> to leave comment</p>
+</div>
+<% end %>
+<% Review.all.where(post_id: session[:id]).each do |review| %>
```

Comment:
`Review.all.where(post_id: session[:id])` should be inside controller
Also `all` here is unnecessary `Review.where` works not worst